spec/models/eligibility_check_spec.rb

```diff
@@ -289,15 +289,15 @@
         { completed_requirements: true, region: create(:region) }
       end
 
-      it { is_expected.to eq(:degree) }
+      it { is_expected.to eq(:qualification) }
     end
 
     context "when degree is present" do
       let(:attributes) do
         { degree: true, completed_requirements: true, region: create(:region) }
       end
 
-      it { is_expected.to eq(:qualification) }
+      it { is_expected.to eq(:teach_children) }
     end
 
     context "when qualification is present" do
```

Comment:
```suggestion
    context "when degree is present" do
```

Then this becomes the test for degree.